src/react-native/04-intro-to-jsx/intro-to-jsx.md

```diff
@@ -39,10 +51,67 @@ If you’ve implemented the solution correctly, the tests will pass when you run
 <details>
 <summary>Click to see the solution</summary>
 
+✏️ Update **src/App.tsx** to be:
+@diff ../../../exercises/react-native/03-creating-a-new-app/04-solution/App.tsx ../../../exercises/react-native/04-intro-to-jsx/01-solution/App.tsx only
+
+</details>
+
+## Objective 2: Expressions and loops in JSX
+
+Next, we want to render a list of states name in our application:
+
+<img alt="Screenshot of a mobile application interface text “Place My Order: Coming Soon To...”, “Illinois“, and “Wisconsin“." src="../../static/img/react-native/04-intro-to-jsx/02-solution.png" style="max-height: 750px;"/>
+
+### Concept TODO
+
 TODO
 
+### Setup 2
+
+✏️ Update **src/App.tsx** to be:
+
+@diff ../../../exercises/react-native/04-intro-to-jsx/01-solution/App.tsx ../../../exercises/react-native/04-intro-to-jsx/02-problem/App.tsx only
```

Comment:
Nothing’s highlighted, is this diff right?

Reply:
That is right, there isnt a setup for these objectives